htdocs/install/mysql/tables/llx_bookmark.sql

```diff
@@ -21,7 +21,7 @@ create table llx_bookmark
   rowid       integer AUTO_INCREMENT PRIMARY KEY,
   fk_user     integer NOT NULL,
   dateb       datetime,
-  url         varchar(255) NOT NULL,
+  url         TEXT,
```

Comment:
Can't you keep the Not null ?

Reply:
I don't know how to do it while keeping PgSQL compatibility. Postgres doesn’t allow NOT NULL without a default value (at least when altering a table), but MySQL doesn’t allow default values for TEXT.